src/modules/deposit/service.ts

```diff
@@ -53,7 +54,9 @@ export class DepositService {
         .createQueryBuilder("d")
         .where("d.status = :status", { status })
         .andWhere("d.depositDate > NOW() - INTERVAL '1 days'")
-        .andWhere("d.depositRelayerFeePct / power(10, 18) >= 0.0001")
+        .andWhere(
+          `d.depositRelayerFeePct / power(10, 18) * ${SUGGESTED_FEES_DEVIATION_BUFFER_MULTIPLIER} >= d.suggestedRelayerFeePct / power(10, 18)`,
```

Comment:
> Does this buffer multiplier make sense? 🤔 I added it because locally I have been seeing some deviations

Hmm, I don't know what to say. What kind of deviations?

Reply:
> Does the division by power(10, 18) need to be done here? The division seems handled on both sides of the comparator, which would cancel it out.

➕

Reply:
> Does the division by power(10, 18) need to be done here? The division seems handled on both sides of the comparator, which would cancel it out.

Thanks makes a lot of sense. Removed it

> Hmm, I don't know what to say. What kind of deviations?

So basically the suggested fees at the time of the deposit deviate from the suggested fees at the time the scraper indexes the deposit sometimes. The multiplier, therefore, acts as a buffer to still consider these deposits as profitable.

Reply:
Hmm, ok. We might want to confirm with Nick that this is ok

Reply:
So why does the "the suggested fees at the time of the deposit deviate from the suggested fees at the time the scraper indexes the deposit " Is it because there is some delay in the scraper? Is this delay on the order of minutes or hours? I would expect some deviation but not big, so maybe 1.1 is too small but definitely not more than say 1.5

Reply:
Yes, because the Scraper has some following distances. Depending on the chain this might be a few minutes but not hours. Of course, there are edge cases like outages, missed events, etc.

Ok, I will make the multiplier configurable and set the default to 1.25